tsconfig.json

```diff
@@ -2,6 +2,7 @@
   "compilerOptions": {
     "typeRoots": ["./types", "./node_modules/@types"],
     "target": "es5",
+    "downlevelIteration": true,
```

Comment:
Why do we need to "downlevel" ?

Reply:
Needed to allow looping iterator objects with es5 rather than just arrays

Reply:
https://www.typescriptlang.org/tsconfig#downlevelIteration